api/top-langs.js

```diff
@@ -26,25 +26,35 @@ module.exports = async (req, res) => {
     layout,
     langs_count,
     exclude_repo,
+    include_repo,
     custom_title,
     locale,
     border_radius,
     border_color,
   } = req.query;
   res.setHeader("Content-Type", "image/svg+xml");
 
+  // Validate input arguments.
   if (blacklist.includes(username)) {
     return res.send(renderError("Something went wrong"));
   }
-
   if (locale && !isLocaleAvailable(locale)) {
     return res.send(renderError("Something went wrong", "Locale not found"));
   }
+  if (include_repo && exclude_repo) {
```

Comment:
@AbhishekPowar I added an error message when both the `include_repo` and `exclude_repo` are specified. @anuraghazra What are your thoughts on adding this error message?